### PR TITLE
[WIP] Added Novem Gold Token (NNN) configuration

### PR DESCRIPTION
**Pull request by corck**
**What current issue(s) from Trello/Github does this address?**

Added NEP5 images and configuration for our Novem Gold Token. 

**How did you make sure your solution works?**

Started in production mode locally. Saw the images appear on the activity page, but not on the wallet page. Some configuration might be missing.

**Is there anything else we should know?**

Please let me know whats missing to have the logo appear on the main wallet page, if that's even the case that it should appear there.

- [ ] Unit tests written?
Not needed.

### CODE REVIEW

**Comment by corck** [comment]
closing as I learned about `neo-tokens` repo just now.

**Comment by comountainclimber** [comment]
@corck Sorry was travelling for devcon when you made this PR and still getting caught up... Currently we are **not leveraging**  the neo-tokens repo (although I still suggest you open a PR there)

Will take a look and test what you have here today and let you know if there are any issues sorry for the delay in getting you feedback on this!

**Comment by corck** [comment]
ah thx.

**Comment by comountainclimber** [comment]
@corck This looks exactly like how things should be set up... After looking things over still not sure why the token would not be loading on the dashboard though - are you sure you had a balance and that the contract has been deployed? I cant seem to find the asset on neoscan

Also, I changed the destination of this PR to our release candidate instead of the `dev`  branch

**Comment by dvd-schwrtz** [comment]
@corck why is the supply 0?

File: app/core/tokenList.json [hunk 1]
```diff
@@ -428,6 +428,19 @@
...
+  "NNN": {
+    "symbol": "NNN",
+    "companyName": "Novem Gold AG",
+    "type": "NEP5",
+    "networks": {
+      "1": {
+        "name": "Novem Gold Token",
+        "hash": "50091057ff12863f1a43266b9786209e399c6ffc",
+        "decimals": 8,
+        "totalSupply": 0
```
Comment (dvd-schwrtz) [inline app/core/tokenList.json]: update supply

**Comment by corck** [comment]
@comountainclimber we haven't minted any coins yet and therefore no transactions so Neoscan has not picked it up yet. You can check against testnet though, can send you some tokens.

just inspected the token balance, where it does not appear (it does on the activities page)
image

not sure why its not rendering the image tag there. don't know react. 

RC branch is great, thx

**Comment by corck** [comment]
@dvd-schwrtz sorry, did not refresh the page and missed your comments.

we mint Gold on the fly when we store it.  Currently we don't have any tokens minted yet, we'll do when we officially launch. also the supply will be changing every other week, so doesn't make sense to hardcode the supply.

File: app/core/tokenList.json (hunk 1 above)
Comment (corck) [reply app/core/tokenList.json]: updated it to 1 and did not resolve my issue.

**Comment by corck** [comment]
I can send you some tokens on testnet if that helps. just ping me an address (I'm on discord also, christophk)

**Comment by comountainclimber** [comment]
@corck going to merge now I reached out on discord... Can debug further once I get some tokens on testnet but everything in this PR looks good to me 👍

**Pull request merged**